src/nerve/core/nodes/terminal/claude_wezterm_node.py

```
@@ -199,6 +211,30 @@ async def create(
+        if mcp_config:
+            import json
+            import tempfile
+
+            # Generate unique filename for MCP config
+            config_filename = f"nerve-mcp-{id}-{uuid_module.uuid4().hex[:8]}.json"
+            mcp_config_path = Path(tempfile.gettempdir()) / config_filename
+
+            # Wrap in mcpServers format (Claude Code expects this structure)
+            full_config = {"mcpServers": mcp_config}
+
+            with open(mcp_config_path, "w") as f:
```

Ruff (0.14.10)

Async functions should not open files with blocking methods like `open`

(ASYNC230)